main.go

```diff
@@ -59,15 +60,10 @@ func main() {
 
 // initAction initializes the frizbee action - reads the environment variables, creates the GitHub client, etc.
 func initAction(ctx context.Context) (*action.FrizbeeAction, error) {
-	// Get the GitHub token from the environment
-	token := os.Getenv("GITHUB_TOKEN")
-	if token == "" {
-		return nil, errors.New("GITHUB_TOKEN environment variable is not set")
-	}
-
-	// Create a new GitHub client
-	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
-	tc := oauth2.NewClient(ctx, ts)
+	var repo *git.Repository
+	var fs billy.Filesystem
+	var githubClient *github.Client
+	var token string
```

Comment:
It looks like a bunch of these (particularly `githubClient` and `token`) don't get initialized on the "directory" path.  Should we remove them?

Reply:
The tricky thing is they're used when creating the `action.FrizbeeAction`, so they need to be defined unconditionally